package.json

```diff
@@ -25,6 +25,7 @@
     "optimist": "",
     "pretty-error": "^2.0.2",
     "purdy": "^2.2.0",
+    "rando-js": "^0.1.1",
```

Comment:
So my only issue with this is that if we're going to go with implementing a random lib I'd prefer that we actually implement something that doesn't use `Math.random` whether it be Mersenne Twister or Alea or whatever I have no opinion